package.json

```diff
@@ -72,6 +72,7 @@
     "js-test-integration-bundle": "rollup --config js/tests/integration/rollup.bundle.js",
     "js-test-integration-modularity": "rollup --config js/tests/integration/rollup.bundle-modularity.js",
     "js-test-cloud": "cross-env BROWSERSTACK=true npm run js-test-karma",
+    "js-test-ltcloud": "cross-env LAMBDATEST=true & node js/tests/lt-local & sleep 5  &  npm run js-test-karma",
```

Comment:
This is is using unix specific features which we generally avoid. Like `&` and `sleep`. For `&` we use `npm-run-all --parallel`. For sleep let's leave it for the time being.